_layouts/blog.html

```diff
@@ -1,16 +1,22 @@
 ---
-layout: default
+layout: page
```

Comment:
Hmmm, if we do that, in order to keep the white-colored links as it is in the landing page, we'd need a dark header background color. The mypaint image logo would not jive with that. Options:

- Use a dark color header so the links can be the same color on every page, and change the mypaint logo into a white text logo to look good with the dark header
- Keep as is to be able to use the mypaint image logo.

In any case, the blue accent color is a placeholder for now (: